src/terminologySectionPanel/codesSection/codesSubSection/codeSubSection/CodeSubSection.tsx

```diff
@@ -13,9 +17,40 @@ export default function CodeSubSection({
   canEdit,
   allCodeSystems,
 }: CodeSectionProps) {
+  const [code, setCode] = useState<Code>();
   const [showResultsTable, setShowResultsTable] = useState(false);
-  const handleFormSubmit = (values) => {
-    setShowResultsTable(true);
+  // toast utilities
+  const [toastOpen, setToastOpen] = useState<boolean>(false);
+  const [toastMessage, setToastMessage] = useState<string>("");
+  const onToastClose = () => {
+    setToastMessage("");
+    setToastOpen(false);
+  };
+
+  const handleFormSubmit = async (values) => {
+    // eslint-disable-next-line
+    const terminologyService = await useTerminologyServiceApi();
```

Comment:
this hook needs some refactoring. it makes requests to get service config each time we use this hook.